libs/core/tsconfig.lib.json

```diff
@@ -5,7 +5,7 @@
     "target": "es2015",
     "declaration": true,
     "inlineSources": true,
-    "types": [],
+    "types": ["node", "jest"],
```

Comment:
@yjaaidi What do you think about it? It's for importing required types in the core package just for building the `@http-ext/core/testing` secondary endpoint. I'm not convinced about it, but didn't found a better solution.

Reply:
🤔interesting! Can't figure out a way to override the tsconfig for `core/testing`.
I guess we should ask Alan Agius from the CLI team.